src/main/java/racinggame/Car.java

```diff
@@ -0,0 +1,21 @@
+package racinggame;
+
+import racinggame.utils.MoveValueValidator;
+
+public class Car {
+    private StringBuilder moveStatus;
+
+    public Car() {
+        this.moveStatus = new StringBuilder();
+    }
+
+    public void advance(int moveValue) {
+        if(MoveValueValidator.validate(moveValue)) {
```

Comment:
```suggestion
        if (MoveValueValidator.validate(moveValue)) {
```
코드 포맷팅 한 번 더 확인해주시면 좋을 것 같습니다.
IDE로 IntelliJ IDEA를 사용하고 계신다면 단축키 `Ctrl + Alt + L`을 활용해보시는 걸 추천드립니다.